src/migratus/cli.clj

```diff
@@ -95,17 +98,109 @@
 
       :else (no-match-message args summary))))
 
+(defn util-date-to-local-datetime [util-date]
+  (if (= util-date nil)
```

Comment:
Use a more readable way to test for nil: 
- https://clojuredocs.org/clojure.core/nil_q  
- https://clojuredocs.org/clojure.core/some_q 

I recommend: https://clojuredocs.org/clojure.core/when-some  .

Reply:
refactored if statement